.eslintignore

```diff
@@ -0,0 +1 @@
+src/intercom.js
```

Comment:
* [ ] Is this an artifact? This file doesn't exist. If I'm not missing somthing, let's revert! ◀️